### ISSUE TITLE
Staging Review finding: Error is not being focused on submit

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#121915]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization - Agile Six Team
**Product:** Marital Status Questionnaire (VA Form 21P-0537)
**Feature:** 21P-0537 Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

- On Step 1 of 4: Your contact details, when clicking continue with errors in the form, focus is not moving to the first field in error. Because of this, screen reader users, or users that are zoomed in, may not be aware of the errors. 
- MacOS/Safari/Voiceover

Duplicate of known issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4993

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to recreate:
- with Voiceover on, navigate to Step 1 of 4: Your contact information
- TAB through the form, past the entry fields, not filling in any data
- TAB to Continue and hit ENTER
- Note behavior
- Other occurence: this also occurs on the Summary screen, when there is an error for Your full name field and the form is subbmitted via keyboard
- Note: may be related to other VO/Safari issues: VO cursor remains on Continue button, programmatic focus is moved to field in error parent

<img width="912" height="660" alt="error not receiving focus" src="https://github.com/user-attachments/assets/cb7d22fb-7743-41ef-ac10-849e7837b9fe" />
---
<img width="765" height="339" alt="Focus not managed on Submit" src="https://github.com/user-attachments/assets/66e9cfa9-5d4b-4940-a367-e8754d4ab809" />

</details>

## Recommended action

Expected behavior:
- Focus moves to the first field in error, and the error is announced accordingly.

https://design.va.gov/templates/forms/accessibility-guidelines#managing-focus-in-form-flows

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [x] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [x] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by michaelclement** [comment]
Confirmed this issue is fixed. @natejones6 Closing as resolved.

|Phone|Signature field|
|-|-|
|<img width="812" height="322" alt="Image" src="https://github.com/user-attachments/assets/1f7e1b32-fa2c-463d-b11c-87e29b067a99" />|<img width="975" height="308" alt="Image" src="https://github.com/user-attachments/assets/4d001e25-edf6-486c-b393-cae83cfcfe4f" />|
